infrastructure/Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: install deploy deploy-control-plane deploy-application-plane deploy-problem destroy synth diff bootstrap build test lint format format-check static-analysis before-commit clean help
+.PHONY: install deploy deploy-control-plane deploy-application-plane deploy-problem destroy destroy-control-plane destroy-application-plane destroy-problem synth diff bootstrap build test lint format format-check static-analysis before-commit clean help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)